packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -211,11 +233,30 @@ const handleOriginRequest = async ({
   const normalisedS3DomainName = normaliseS3OriginDomain(s3Origin);
   const hasFallback = hasFallbackForUri(uri, prerenderManifest);
   const { now, log } = perfLogger(manifest.logLambdaExecutionTimes);
+  const isPreviewRequest = getPreviewCookies(request);
+
+  if (isPreviewRequest) {
+    try {
+      jsonwebtoken.verify(
+        isPreviewRequest[NEXT_PREVIEW_DATA_COOKIE],
+        prerenderManifest.preview.previewModeSigningKey
+      );
+    } catch (e) {
+      return {
```

Comment:
Maybe some logging is useful here, I added it for 500 page as well, not sure if it's needed for this though

Reply:
I added a `console.error`, similar to the logging for the 500 response.